super_team_picker_4/routes/cohorts.js

```diff
@@ -0,0 +1,109 @@
+const express = require('express');
+const knex = require('../db/client');
+const router = express.Router();
+const teamMethod = require('../public/JS/shuffle')
+
+
+
+router.get('/', (req, res) => {
+  knex("cohorts")
+  .orderBy('createdAt', 'desc')
+  .then(data => {
+    console.log(data);
+    res.render('cohorts/index', {list:data})
+  })
+})
+
+router.post('/', (req, res) => {
+  knex("cohorts")
+  .insert({
+    name: req.body.name,
+    member: req.body.member,
+    logoUrl: req.body.logoUrl
+  })
+  .returning('*')
+  .then(data => {
+    res.redirect(`/cohorts/${data[0].id}`)
+  })
+})
+
+router.get('/new', (req, res) => {
+    res.render('cohorts/new')
+})
+
+router.get('/:id', (req, res) => {
+  const id = req.params.id;
+  let teams = []
+  knex("cohorts")
+  .where("id", id)
+  .first()
+  .then(data => {
+    let checkTeam = null
+    let checkQuantity = null
+    if (req.query.quantity) {
+      let names = data.member.split(',')
+      let quantity = parseInt(req.query.quantity)
+      console.log(names, quantity);
```

Comment:
clean up any logs you had for dev work before submission unless they are absolutely needed